pages/profile/index.js

```diff
@@ -21,7 +21,11 @@ module.exports = () => {
     }
     return req.api(`/establishment/${req.establishment}/profile/${profile}`)
       .then(({ json: { data, meta } }) => {
-        req.model = cleanModel(data);
+        const model = cleanModel(data);
+        const modules = model.trainingModules;
+        model.exemptions = filter(modules, (m) => { return m.exemption; });
+        model.trainingModules = filter(modules, (m) => { return !m.exemption; });
```

Comment:
I'm guessing these are arrays? If so let's use the vanilla `Array.prototype.filter`